src/pom.xml

```diff
@@ -1948,7 +1948,7 @@
   <wicket.version>7.6.0</wicket.version>
   <ant.version>1.8.4</ant.version>
   <imageio-ext.version>1.2.0</imageio-ext.version>
-  <jaiext.version>1.1.4</jaiext.version>
+  <jaiext.version>1.1-SNAPSHOT</jaiext.version>
```

Comment:
we may want to update to 1.1.next once new jai-ext is released

Reply:
Yep, that's why I said it depends on a jai-ext release :-)